SteamReview/SteamReview/Game.cpp

```diff
@@ -9,22 +9,38 @@ FGame::FGame()
 };
```

Comment:
I see you added a commit with changes for David's comments. But some of them have not been addressed (like this one). If you don't agree with something or don't understand something, it's good to leave a comment about it, rather than ignoring it.

Or in the case that you guys discussed this in person and decided to not apply the comment, it's still good to leave the conclusion written in the review.

However in this case, I do agree with David :) It increases readability and consistency of whitespace.